openverse_catalog/docs/provider_data_ingester_faq.md

```diff
@@ -0,0 +1,142 @@
+# ProviderDataIngester FAQ
+
+The most straightforward implementation of a `ProviderDataIngester` repeats the following process:
+
+* Builds a set of query params for the next request, based upon the previous params (for example, by updating offsets or page numbers)
+* Makes a single GET request to the configured static `endpoint` using the built params
+* Extracts a "batch" of records from the response, as a list of record JSON representations
+* Iterates over the records in the batch, extracting desired data, and commits them to local storage
+
+Some provider APIs may not fit neatly into this workflow. This document addresses some common use cases.
+
+## How do I process a provider "record" that contains data about multiple records?
+
+**Example**: You're pulling data from a Museum database, and each "record" in a batch contains multiple photos of a single physical object.
+
+**Solution**: The `get_record_data` method takes a `data` object representing a single record from the provider API. Typically, it extracts required data and returns it as a single dict. However, it can also return a **list of dictionaries** for cases like the one described, where multiple Openverse records can be extracted.
```

Comment:
It would be nice to add links to the examples of each of the solutions.

Reply:
I wanted to do this but was worried about code drift and making this document difficult to maintain 🤔

Reply:
Ooo, that's a really good point!

Reply:
The problem with code drift and maintainability is really a good point :( I wonder if we could automate it somehow, add a link to some regex search string that would lead to the example of, say, overridden method in one of the solutions...  

Not necessary for this PR, but would be really nice to have. I usually learn by example and links to the examples of the solutions would be really helpful for the way _I_ learn :)